ui/src/routes/Subjects/Subjects.js

```diff
@@ -1,14 +1,33 @@
 import _ from 'lodash'
 import React, { Component } from 'react'
 import { browserHistory } from 'react-router'
+import { Icon } from 'react-fa'
 import BreadcrumbChevron from '../../components/BreadcrumbChevron'
+import PageFocusSection from '../../components/PageFocusSection'
 import { log } from '../../utilities'
 
 import '../../styles/components/c-breadcrumbs.css'
 
 let backgroundImage = require('../../assets/clix-i2c-flowers.svg')
 
 class Subjects extends Component {
+  constructor (props) {
+    super(props)
+    this.state = {
+      skiplinks: [{
```

Comment:
Do you need to define `skipLinks` here? I think it's already defined in `PageFocusSection`...